Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2522,7 +2522,7 @@ HRESULT Stage(CGContextRef context, ID2D1DeviceContext* deviceContext) override
             deviceContext->Clear({ 0, 0, 0, 0 }); // Clear the original target to transparent black.
             RETURN_IF_FAILED(inputBitmap.As(&copiedImage));
         }
-        return __super::Stage(context, deviceContext);
+        return __super ::Stage(context, deviceContext);
```

Comment:
now I just wish there was a simpler way to do this.

---
In reply to: [106458788](https://github.com/Microsoft/WinObjC/pull/2249#discussion_r106458788) [](ancestors = 106458788)